File: Python/PRAK305-2210817210015-MuhammadIrfan.py
sec = int(input("Masukkan jumlah detik: "))
hari = sec // (60*60*24)
modA = sec % (60*60*24)
jam = modA // (60*60)
modB = sec % (60*60)
menit = modB // 60
detik = sec % 60
if sec>=1 and sec<60:
    print("00:00:{}" .format(detik))
elif sec>=60 and sec<3600:
    print("00:{}:{}" .format(menit, detik))
elif sec>=3600 and sec<86400:
    print("{}:{}:{}" .format(jam, menit, detik))
else:
    print("{} hari {}:{}:{}" .format(hari, jam, menit, detik))

File: C/PRAK305-2210817210015-MuhammadIrfan.c
#include<stdio.h>
    int main(){
        int sec, hari, jam, menit, detik, modA, modB;
        printf("Masukkan jumlah detik: "); scanf("%d", &sec);
        hari = sec / (60*60*24);
        modA = sec % (60*60*24);
        jam = modA / (60*60);
        modB = sec % (60*60);
        menit = modB / 60;
        detik = sec % 60;
        if (sec>=1 && sec<60){
            printf("00:00:%d", detik);}
            else if (sec>=60 && sec<3600){
            printf("00:%d:%d", menit, detik);}
            else if (sec>=3600 && sec<86400){
            printf("%d:%d:%d", jam, menit, detik);}
            else{
            printf("%d hari %d:%d:%d", hari, jam, menit, detik);}
        return 0;
    }